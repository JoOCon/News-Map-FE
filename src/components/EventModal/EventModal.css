@font-face {
  font-family: caviar;
  src: url("../../fonts/CaviarDreams.ttf"); }

@font-face {
  font-family: caviar-bold;
  src: url("../../fonts/CaviarDreams_Bold.ttf"); }

.fas,
.fa-eye {
  font-size: 2rem;
  color: #5e7676;
  cursor: pointer; }

.fa-eye:hover {
  color: darkslategray; }

.far {
  color: #888888; }

.hover-msg {
  font-size: 1rem;
  color: #838383; }

.event-modal-container {
  font-family: caviar; }

.event-name {
  font-family: caviar-bold;
  color: #114211; }

.event-venue {
  font-size: 1.4rem;
  font-weight: 800; }

.event-address {
  font-size: 1.2rem; }

.styles_modal__gNwvD {
  border-radius: 1rem;
  background-color: rgba(225, 225, 225, 0.927); }

.event-img {
  border-radius: 10rem;
  filter: grayscale(50%); }

.styles_closeIcon__1QwbI {
  cursor: pointer; }
