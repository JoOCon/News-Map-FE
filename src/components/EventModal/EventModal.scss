.fas,
.fa-eye {
  font-size: 2rem;
  color: darkslategray;
}

.far {
  color: rgb(136, 136, 136);
}

.hover-msg {
  font-size: 1rem;
  color: rgb(131, 131, 131);
}
