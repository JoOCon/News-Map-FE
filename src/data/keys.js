export const googleClientId =
  '241779944002-qa42v6cvk3kpu8da0jimdr364vn41t1r.apps.googleusercontent.com';
