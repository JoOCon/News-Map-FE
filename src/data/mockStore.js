import * as user from './mockUser';
import * as events from './mockEvents';

export const mockStore = {
  activeUser: user.mockStateUser,
  userLocation: { latitude: 39.43243, longitude: -104.98434 },
  watchList: events.userWatchlist,
  events: events.eventsResponse
};
