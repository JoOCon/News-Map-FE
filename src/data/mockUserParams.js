export const responseGoogle1 =
  'https://event-mapper-api.herokuapp.com/api/v1/users';

export const responseGoogle2 = {
  body:
    '{"user":{"google_id":"117977763637801404146","given_name":"Paul","family_name":"K","email":"pykim0591@gmail.com"}}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const getEvents =
  'https://app.ticketmaster.com/discovery/v2/events.json?apikey=GddBLHQGZLJjPNWsNxOrYRAoBx8Hdqrf&geoPoint=9wjr4et3f&radius=200&size=100';

export const postUser = {
  body: '{"user":{"email":"pykim0591@gmail.com"}}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const setFavorite = {
  body: '{}',
  headers: { 'Content-Type': 'application/json' },
  method: 'POST'
};

export const removeEvent =
  'https://event-mapper-api.herokuapp.com/api/v1/users/undefined/events/undefined';

export const removeEventOptions = {
  headers: { 'Content-Type': 'application/json' },
  method: 'DELETE'
};
