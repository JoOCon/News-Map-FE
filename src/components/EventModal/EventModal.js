import React from 'react';
import PropTypes from 'prop-types';
import './EventModal.css';

export const EventModal = ({ targetEvent, handleFavoriteClick, handleHover, hoverMessage }) => {
  const { name, img, date, venue_name, favorite, address } = targetEvent;
  return (
    <div className="event-modal-container">
      {!favorite ? (
        <i
          className="fas fa-eye"
          onClick={handleFavoriteClick}
          onMouseEnter={event => handleHover(event, 'Add to Watch-list')}
          onMouseLeave={event => handleHover(event, '')}
        >
          <span className="hover-msg">{hoverMessage}</span>
        </i>
      ) : (
        <i
          className="far fa-eye"
          onClick={handleFavoriteClick}
          onMouseEnter={event => handleHover(event, 'Remove from Watch-list')}
          onMouseLeave={event => handleHover(event, '')}
        >
          <span className="hover-msg">{hoverMessage}</span>
        </i>
      )}
      <h1 className="event-name">{name}</h1>
      <h4 className="event-date">{date}</h4>
      <img
        src={`${img}`}
        alt="current event"
        height={400}
        className="event-img"
      />
      <p className="event-venue">{venue_name}</p>
      <p className="event-address">{address}</p>
    </div>
  );
};

EventModal.propTypes = {
  targetEvent: PropTypes.object, 
  handleFavoriteClick: PropTypes.func, 
  handleHover: PropTypes.func, 
  hoverMessage: PropTypes.string
};