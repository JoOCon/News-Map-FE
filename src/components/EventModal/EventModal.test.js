import React from 'react';

import { EventModal } from './EventModal';
import * as events from '../../data/mockEvents';

describe('EventModal', () => {
  let wrapper;
  let mockFavoriteClick;
  let mockHandleHover;
  let mockMessage;

  beforeEach(() => {
    mockFavoriteClick = jest.fn();
    mockHandleHover = jest.fn();
    mockMessage = 'help me';
    wrapper = shallow(
      <EventModal
        targetEvent={events.mockTargetEventFalse}
        handleFavoriteClick={mockFavoriteClick}
        handleHover={mockHandleHover}
        hoverMessage={mockMessage}
      />
    );
  });

  it('should matchSnapshot when favorite is false', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should matchSnapshot when favorite is true', () => {
    wrapper = shallow(
      <EventModal
        targetEvent={events.mockTargetEventTrue}
        handleFavoriteClick={mockFavoriteClick}
        handleHover={mockHandleHover}
        hoverMessage={mockMessage}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });

  it('should call handleHover on mouse enter and mouse leave when favorite is false', () => {
    const mockEvent = { preventDefault: jest.fn() };

    wrapper.find('i').simulate('mouseEnter', mockEvent);
    expect(mockHandleHover).toHaveBeenCalledWith(
      mockEvent,
      'Add to Watch-list'
    );

    wrapper.find('i').simulate('mouseLeave', mockEvent);

    expect(mockHandleHover).toHaveBeenCalledWith(mockEvent, '');
  });

  it('should call handleHover with correct params on mouse enter and mouse leave when favorite is true', () => {
    wrapper = shallow(
      <EventModal
        targetEvent={events.mockTargetEventTrue}
        handleFavoriteClick={mockFavoriteClick}
        handleHover={mockHandleHover}
        hoverMessage={mockMessage}
      />
    );
    const mockEvent = { preventDefault: jest.fn() };

    wrapper.find('i').simulate('mouseEnter', mockEvent);
    expect(mockHandleHover).toHaveBeenCalledWith(
      mockEvent,
      'Remove from Watch-list'
    );

    wrapper.find('i').simulate('mouseLeave', mockEvent);
    expect(mockHandleHover).toHaveBeenCalledWith(mockEvent, '');
  });
});
