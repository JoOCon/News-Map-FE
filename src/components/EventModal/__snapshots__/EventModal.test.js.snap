// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventModal should matchSnapshot when favorite is false 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventModal
    handleFavoriteClick={[MockFunction]}
    handleHover={[MockFunction]}
    hoverMessage="help me"
    targetEvent={
      Object {
        "address": "1510 Clarkson Denver, CO 80218",
        "date": "Wed, Oct 31, 2018 6:30 PM",
        "distance": 0.5,
        "e_id": "G5vzZ4C58cdb-",
        "favorite": false,
        "img": "https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg",
        "lat": "39.740611",
        "lng": "-104.977194",
        "name": "Danzig - 30 Year Anniversary Tour",
        "url": "http://concerts.livenation.com/event/1E0054E2D0B43621",
        "venue_name": "Fillmore Auditorium (Denver)",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <i
          className="fas fa-eye"
          onClick={[MockFunction]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <span
            className="hover-msg"
          >
            help me
          </span>
        </i>,
        <h1>
          Event: 
          Danzig - 30 Year Anniversary Tour
        </h1>,
        <h4>
          Date: 
          Wed, Oct 31, 2018 6:30 PM
        </h4>,
        <img
          alt="current event"
          height={400}
          src="https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg"
        />,
        <p>
          Location: 
          Fillmore Auditorium (Denver)
        </p>,
        <p>
          Address: 
          1510 Clarkson Denver, CO 80218
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <span
            className="hover-msg"
          >
            help me
          </span>,
          "className": "fas fa-eye",
          "onClick": [MockFunction],
          "onMouseEnter": [Function],
          "onMouseLeave": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "help me",
            "className": "hover-msg",
          },
          "ref": null,
          "rendered": "help me",
          "type": "span",
        },
        "type": "i",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Event: ",
            "Danzig - 30 Year Anniversary Tour",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Event: ",
          "Danzig - 30 Year Anniversary Tour",
        ],
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Date: ",
            "Wed, Oct 31, 2018 6:30 PM",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Date: ",
          "Wed, Oct 31, 2018 6:30 PM",
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "current event",
          "height": 400,
          "src": "https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Location: ",
            "Fillmore Auditorium (Denver)",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Location: ",
          "Fillmore Auditorium (Denver)",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Address: ",
            "1510 Clarkson Denver, CO 80218",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Address: ",
          "1510 Clarkson Denver, CO 80218",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <i
            className="fas fa-eye"
            onClick={[MockFunction]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <span
              className="hover-msg"
            >
              help me
            </span>
          </i>,
          <h1>
            Event: 
            Danzig - 30 Year Anniversary Tour
          </h1>,
          <h4>
            Date: 
            Wed, Oct 31, 2018 6:30 PM
          </h4>,
          <img
            alt="current event"
            height={400}
            src="https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg"
          />,
          <p>
            Location: 
            Fillmore Auditorium (Denver)
          </p>,
          <p>
            Address: 
            1510 Clarkson Denver, CO 80218
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <span
              className="hover-msg"
            >
              help me
            </span>,
            "className": "fas fa-eye",
            "onClick": [MockFunction],
            "onMouseEnter": [Function],
            "onMouseLeave": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "help me",
              "className": "hover-msg",
            },
            "ref": null,
            "rendered": "help me",
            "type": "span",
          },
          "type": "i",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Event: ",
              "Danzig - 30 Year Anniversary Tour",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Event: ",
            "Danzig - 30 Year Anniversary Tour",
          ],
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Date: ",
              "Wed, Oct 31, 2018 6:30 PM",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Date: ",
            "Wed, Oct 31, 2018 6:30 PM",
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "current event",
            "height": 400,
            "src": "https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Location: ",
              "Fillmore Auditorium (Denver)",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Location: ",
            "Fillmore Auditorium (Denver)",
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Address: ",
              "1510 Clarkson Denver, CO 80218",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Address: ",
            "1510 Clarkson Denver, CO 80218",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`EventModal should matchSnapshot when favorite is true 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventModal
    handleFavoriteClick={[MockFunction]}
    handleHover={[MockFunction]}
    hoverMessage="help me"
    targetEvent={
      Object {
        "address": "1510 Clarkson Denver, CO 80218",
        "date": "Wed, Oct 31, 2018 6:30 PM",
        "distance": 0.5,
        "e_id": "G5vzZ4C58cdb-",
        "favorite": true,
        "img": "https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg",
        "lat": "39.740611",
        "lng": "-104.977194",
        "name": "Danzig - 30 Year Anniversary Tour",
        "url": "http://concerts.livenation.com/event/1E0054E2D0B43621",
        "venue_name": "Fillmore Auditorium (Denver)",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <i
          className="far fa-eye"
          onClick={[MockFunction]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        >
          <span
            className="hover-msg"
          >
            help me
          </span>
        </i>,
        <h1>
          Event: 
          Danzig - 30 Year Anniversary Tour
        </h1>,
        <h4>
          Date: 
          Wed, Oct 31, 2018 6:30 PM
        </h4>,
        <img
          alt="current event"
          height={400}
          src="https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg"
        />,
        <p>
          Location: 
          Fillmore Auditorium (Denver)
        </p>,
        <p>
          Address: 
          1510 Clarkson Denver, CO 80218
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <span
            className="hover-msg"
          >
            help me
          </span>,
          "className": "far fa-eye",
          "onClick": [MockFunction],
          "onMouseEnter": [Function],
          "onMouseLeave": [Function],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "help me",
            "className": "hover-msg",
          },
          "ref": null,
          "rendered": "help me",
          "type": "span",
        },
        "type": "i",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Event: ",
            "Danzig - 30 Year Anniversary Tour",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Event: ",
          "Danzig - 30 Year Anniversary Tour",
        ],
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Date: ",
            "Wed, Oct 31, 2018 6:30 PM",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Date: ",
          "Wed, Oct 31, 2018 6:30 PM",
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "current event",
          "height": 400,
          "src": "https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Location: ",
            "Fillmore Auditorium (Denver)",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Location: ",
          "Fillmore Auditorium (Denver)",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Address: ",
            "1510 Clarkson Denver, CO 80218",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Address: ",
          "1510 Clarkson Denver, CO 80218",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <i
            className="far fa-eye"
            onClick={[MockFunction]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
          >
            <span
              className="hover-msg"
            >
              help me
            </span>
          </i>,
          <h1>
            Event: 
            Danzig - 30 Year Anniversary Tour
          </h1>,
          <h4>
            Date: 
            Wed, Oct 31, 2018 6:30 PM
          </h4>,
          <img
            alt="current event"
            height={400}
            src="https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg"
          />,
          <p>
            Location: 
            Fillmore Auditorium (Denver)
          </p>,
          <p>
            Address: 
            1510 Clarkson Denver, CO 80218
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <span
              className="hover-msg"
            >
              help me
            </span>,
            "className": "far fa-eye",
            "onClick": [MockFunction],
            "onMouseEnter": [Function],
            "onMouseLeave": [Function],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "help me",
              "className": "hover-msg",
            },
            "ref": null,
            "rendered": "help me",
            "type": "span",
          },
          "type": "i",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Event: ",
              "Danzig - 30 Year Anniversary Tour",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Event: ",
            "Danzig - 30 Year Anniversary Tour",
          ],
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Date: ",
              "Wed, Oct 31, 2018 6:30 PM",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Date: ",
            "Wed, Oct 31, 2018 6:30 PM",
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "current event",
            "height": 400,
            "src": "https://s1.ticketm.net/dam/a/492/732ca6af-f94f-477d-9e66-5f56f9e54492_450551_EVENT_DETAIL_PAGE_16_9.jpg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Location: ",
              "Fillmore Auditorium (Denver)",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Location: ",
            "Fillmore Auditorium (Denver)",
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Address: ",
              "1510 Clarkson Denver, CO 80218",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Address: ",
            "1510 Clarkson Denver, CO 80218",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
